src/server/scripts/Spells/spell_generic.cpp

```diff
@@ -48,6 +48,11 @@
 //  see: https://github.com/azerothcore/azerothcore-wotlk/issues/9766
 #include "GridNotifiersImpl.h"
 
+enum broadcastTextIDs
+{
+    BROADCAST_TEXT_FLAG_OF_OWNERSHIP     = 28008, // The Flag of Ownership
```

Comment:
This comment is redundant (but appreciated).

Reply:
Would you like it removed? I'm unsure due to the appreciation. I can do so once I know a solution to the TextEmote issue. Thanks!

Reply:
It should be removed IMO, yes.